code/modules/shuttle/shuttle_manipulator.dm

```diff
@@ -57,6 +59,11 @@
 /obj/machinery/shuttle_manipulator/attack_hand(mob/user)
 	ui_interact(user)
 
+/obj/machinery/shuttle_manipulator/vv_edit_var(var_name, var_value)
+	switch(var_name)
+		if("shuttle_and_preview_cooldown")
+			log_and_message_admins("has attempted to change the [var_name] variable. Please do not do this, this can cause entire Z levels to freeze if spammed too quickly.")
+			return FALSE /// Extremely important that this doesn't get varedited by mistake, otherwise horrible, horrible things can happen to the server.
```

Comment:
```suggestion
			return FALSE // Extremely important that this doesn't get varedited by mistake, otherwise horrible, horrible things can happen to the server.
```

Reply:
Done